.gitignore

```diff
@@ -18,6 +18,7 @@
 /Library/PinnedKegs
 /Library/PinnedTaps
 /Library/Taps
+/Library/lock.d
```

Comment:
This also feels like it should live in either `tmp` or wherever Homebrew's existing locks live.